google-credentials/.gitignore

```diff
@@ -0,0 +1 @@
+key.json
```

Comment:
i'd put this into the root .gitignore

Reply:
then i need a keep file in this folder, or should we just ignore whole folder?